rainier-core/src/main/scala/com/stripe/rainier/core/RandomVariable.scala

```diff
@@ -58,6 +58,27 @@ class RandomVariable[+T](val value: T, val targets: Set[Target]) {
     sampledParams.map(fn)
   }
 
+  def toSample()(implicit rng: RNG): Sample[T] =
```

Comment:
I think the alternative would be a `Sample.apply(rv, ...)`. Not sure there's any reason to prefer one over the other.

Reply:
Yeah, I considered that form, I think this is more in keeping with the API as it exists currently.